torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16126,7 +16126,12 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                #               'tensors' but instead found type 'Tensor (inferred)'.
                DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_jit_alias_remapping'),
                # see https://github.com/pytorch/pytorch/issues/71286
-               DecorateInfo(unittest.expectedFailure, 'TestNNCOpInfo', 'test_nnc_correctness'),)),
+               DecorateInfo(unittest.expectedFailure, 'TestNNCOpInfo', 'test_nnc_correctness'),
+               # Previously skipped here, changed to xfail:
+               # https://github.com/pytorch/pytorch/blob/e63c502baa4a6f2109749984be701e722b3b7232/torch/testing/_internal/common_utils.py#L4371-L4372
```

Comment:
nit: We probably don't need a comment about what changed (previously skipped where), could you file an issue about what sample input this is failing for? (we could put the comment about all the tests beign skipped previously there instead)

Reply:
Track here: #99806

Reply:
Replaced the comment with the issue link